hubgrep/frontend_blueprint/templates/search/search_base.html

```diff
@@ -16,8 +16,8 @@
         <div class="search-result">
             <div class="search-result-messages">
                 {% import "components/status_message/status_message.html" as status %}
```

Comment:
makes this a bit more text - but its getting the label from the hosting_service_interface and the error from the response...

Reply:
(why is this comment on the other side?)